apps/bluetooth/app.js

```diff
@@ -507,13 +511,13 @@ module.exports = function (activity) {
     }
     switch (state) {
       case protocol.DISCOVERY_STATE.ON:
-        if (lastIntent === 'bluetooth_broadcast') {
+        if (lastIntent !== 'implicit_disconnect' && lastIntent !== 'bluetooth_disconnect') {
```

Comment:
Should this be an `and` op?

Reply:
Yes, lastIntent `not equal` xxx `and` lastIntent `not equal` yyy.